prihlas-me.php

```diff
@@ -0,0 +1,23 @@
+<?php
+session_start();
+
+$uzivatelskeJmeno = '';
+$uzivatelskeHeslo = '';
+
+// pokud neexistuje session, zalozime si ji
+if (!isset($_SESSION['pokusu_prihlaseni'])) {
```

Comment:
Většinou používáme `empty`. Minule už sice Petr `isset` použil, ale pak to stejně vracel na `empty`. Takže buď bych použil to a nebo jim `isset` trochu představil.